widgets/src/Learn/Page.jsx

```diff
@@ -360,6 +468,13 @@ return (
         you where you need it most
       </h2>
     </Heading>
+    <PartnerSection>
+    <h3>NEAR Horizon x Encode Club Educate Series</h3>
+    <Widget
+      src={`humanman.near/widget/Learn.List.Partner`}
+      props={{ cardData: props.encodeEducate }}
+    />
```

Comment:
Is there a reason you are using your own widget instead of creating a new one for the Horizon account?
If not it makes sense to use the Horizon account here, and add the code to that widget in this repo